src/providers/VoiceFocusProvider/index.tsx

```diff
@@ -165,21 +175,24 @@ const VoiceFocusProvider: React.FC<Props> = ({ spec, options, children }) => {
   async function initVoiceFocus(
     vfSpec: VoiceFocusSpec | undefined,
     options: VoiceFocusDeviceOptions | undefined,
+    createMeetingResponse: any | undefined,
     canceled: () => boolean
   ) {
     // Throw away the old one and reinitialize.
     setVoiceFocusTransformer(null);
     setVoiceFocusDevice(null);
-    createVoiceFocusDeviceTransformer(vfSpec, options, canceled);
+    createVoiceFocusDeviceTransformer(vfSpec, options, createMeetingResponse, canceled);
   }
 
   useEffect(() => {
     let canceled = false;
-    initVoiceFocus(vfSpec, options, () => canceled);
+    if (createMeetingResponse) {
```

Comment:
no will only initVoiceFocus when there is createMeetingResponse, because only when createMeetingResponse is defined we could see if the transformer is VF or RES.

Reply:
So if we have no createMeetingResponse, then we dont need to initVoiceFocus?

Reply:
So, this means if the builders wont provide the `createMeetingResponse` the voice focus wont be supported which will be a backward incompatible change. We want to support both:
1. Builders should be able to continue VF use without ER. If this is not supported with this logic then it will be a breaking change. Builders will upgrade to the new React component library version and their app will break if they are using VFProvider cause, now the init needs `createMeetingResponse`.
2. Builder should be able to use ER with VF.